src/firefly/js/tables/TableUtil.js

```diff
@@ -120,7 +120,7 @@ export function makeIrsaCatalogRequest(title, project, catalog, params={}, optio
     title = title || catalog;
     options.use = options.use || 'catalog_overlay';
     const tbl_id = options.tbl_id || uniqueTblId();
-    const id = 'GatorQuery';
+    const id = params.processorName||'GatorQuery';
```

Comment:
Thanks Loi's explanation that makeIrsaCatalogRequest is specified for GatorQuery.  The TableUntil is revert to the dev version.  The corresponding changes are made in Finderchart.jsx.  The branch " IRSA-522-Finderchart" is under ife  located in the server. 
Please verify it again.  Thanks!